test/src/api/responderTest.php

```diff
@@ -4767,6 +4768,8 @@ public function test_interface_game_009() {
         // now load the game and check its state
         $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
 
+        // the courtesy die is no longer offered after this
+        $expData['playerDataArray'][1]['swingRequestArray'] = array();
```

Comment:
Okay, so, a side point here: the flow of logic in responderTest should be:

1. Take an action and make sure it makes sense
2. Modify `$expData` to match the new state of the world after the action
3. use `verify_api_loadGameData()` to test `$expData`

Here you're modifying `$expData` (item 2) *before* taking the action (item 1), which makes it unnecessarily difficult to see what's going on.